src/ha/panels/lovelace/editor/structs/action-struct.ts

```diff
@@ -41,6 +41,7 @@ const actionConfigStructService = object({
       entity_id: optional(union([string(), array(string())])),
       device_id: optional(union([string(), array(string())])),
       area_id: optional(union([string(), array(string())])),
+      label_id: optional(union([string(), array(string())])),
```

Comment:
```suggestion
      floor_id: optional(union([string(), array(string())])),
      label_id: optional(union([string(), array(string())])),
```

Floor can be added too.

Reply:
I don't have floors, that's why they don't show up on me. Thanks for the head up and the fix :)